test/stdlib/Accelerate.swift

```diff
@@ -1,22 +1,27 @@
 // RUN: %target-run-simple-swift
 // REQUIRES: executable_test
 
-// UNSUPPORTED: OS=watchos
```

Comment:
This one caused [problems in master](https://github.com/apple/swift/pull/11580).